<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngKLoginDialog</class>
 <widget class="QWidget" name="AngKLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AngKLoginDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="chooseWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>115</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="logoTitle">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string>Select mode and device</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>50</number>
           </property>
           <property name="leftMargin">
            <number>211</number>
           </property>
           <property name="topMargin">
            <number>39</number>
           </property>
           <property name="rightMargin">
            <number>211</number>
           </property>
           <property name="bottomMargin">
            <number>110</number>
           </property>
           <item>
            <widget class="AngKLoginChoose" name="manualWidget" native="true"/>
           </item>
           <item>
            <widget class="AngKLoginChoose" name="autoWidget" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="loginWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="AngKLoginInput" name="loginInput" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AngKLoginChoose</class>
   <extends>QWidget</extends>
   <header>angkloginchoose.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AngKLoginInput</class>
   <extends>QWidget</extends>
   <header>angklogininput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
